src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -142,17 +144,26 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a fasta file from a hvc
                 //These are optional header fields so we check these in the unit test.
```

Comment:
I agree, I made a new trello card for this as it will take a bit of work to make sure the types all match up and the validation that happens currently will still happen.  Should be a relatively minor change.